controllers/ProfileController.js

```diff
@@ -36,8 +36,51 @@ const getUserLikes = async (req, res) => {
 
 const getProfileContents = async (req, res) => {
   try {
-    const { username } = req.query;
+    const { userId, username } = req.query;
     const query = await pool.query(profileQueries.getProfileContents, [
+      userId,
+      username,
+    ]);
+    res.send(query.rows[0]);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const getFollowStatus = async (req, res) => {
+  try {
+    const { userId, username } = req.query;
+    const query = await pool.query(profileQueries.getFollowStatus, [
+      userId,
+      username,
+    ]);
+    res.send(query.rows[0]);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const followUser = async (req, res) => {
+  try {
+    const { userId, username } = req.body;
+    const query = await pool.query(profileQueries.followUser, [
+      userId,
+      username,
+    ]);
+    res.send(query.rows[0]);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const unfollowUser = async (req, res) => {
+  try {
+    const { userId, username } = req.body;
+    const query = await pool.query(profileQueries.unfollowUser, [
+      userId,
```

Comment:
Alright this means I will update `getUserPosts`, `getUserReplies`, `getUserLikes`, `followUser` and `unFollowUser` to utilize the userId of the visited user instead of the username. I will do this to try and be consistent with taking the userId as a priority whenever we need to identify the user